logger/logger.go

```diff
@@ -0,0 +1,46 @@
+package logger
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/rifflock/lfshook"
+	"github.com/sirupsen/logrus"
+)
+
+var Log *logrus.Logger
+
+func NewLogger() *logrus.Logger {
+	if Log != nil {
+		return Log
+	}
+	var flag string
+	var filePath string
+	fmt.Println("Enter Path to log debug files: (Y/N)")
+	fmt.Scan(&flag)
+	if flag == "N" {
+		id := uuid.New().String()
+		filePath = "debug/Track-" + id + ".log"
+		fileName := "Track-" + id + ".log"
+		fmt.Printf("New File \"%s\" created in debug folder\n", fileName)
```

Comment:
Where are all these logs going to get printed? THe user visible logs should only be sent via the logger, not directly like this.

Reply:
Only this log, Which is showing new file created is printed directly, since we are in logger initialization only and haven't return the logger. Apart from this every log is getting printed via logger.

Reply:
but it is not necessary for this one to be printed directly. The user doesn't need to know what the debug log file is. They only need it if something fails, then they can be told "for details see log here"